docs/image_formats_support.md

```diff
@@ -20,55 +20,55 @@ At the moment, imgproxy supports only the most popular image formats:
 
 ## GIF support
 
-imgproxy supports GIF output only when using libvips 8.7.0+ compiled with ImageMagick support. Official imgproxy Docker image supports GIF out of the box.
+imgproxy supports GIF output only when using libvips 8.7.0+ compiled with ImageMagick support. The official imgproxy Docker image supports GIF output out of the box.
```

Comment:
We need to remove "GIF support", "ICO support", "HEIC support", "AVIF support", and "BMP support" sections as they are not applicable to v3